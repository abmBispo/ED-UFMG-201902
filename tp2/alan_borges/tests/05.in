1000 10 3
7 tul
52 xvf
209 uab
351 svk
371 mvo
340 nbc
307 dmw
37 jko
78 qii
116 txi