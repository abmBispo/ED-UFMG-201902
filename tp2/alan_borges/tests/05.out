1 dmw 307
1 jko 37
1 qii 78
1 tul 7
1 txi 116
1 uab 209
1 xvf 52
2 nbc 340
2 svk 351
3 mvo 371
